addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/api/FreeboxApiManager.java

```diff
@@ -0,0 +1,572 @@
+    }
+
+    public int decreaseLcdBrightness() throws FreeboxException {
+        FreeboxLcdConfig config = getLcdConfig();
+        config.setBrightness(Math.max(0, config.getBrightness() - 1));
+        return executeUrl("PUT", "lcd/config/", gson.toJson(config), LcdConfigResponse.class).getBrightness();
+    }
+
+    public FreeboxLcdConfig setLcdOrientation(int orientation) throws FreeboxException {
+        FreeboxLcdConfig config = getLcdConfig();
+        int newValue = Math.min(360, orientation);
+        newValue = Math.max(newValue, 0);
+        config.setOrientation(newValue);
+        config.setOrientationForced(true);
+        return executeUrl("PUT", "lcd/config/", gson.toJson(config), LcdConfigResponse.class);
+    }
+
+    public boolean setLcdOrientationForced(boolean forced) throws FreeboxException {
+        FreeboxLcdConfig config = getLcdConfig();
+        config.setOrientationForced(forced);
+        return executeUrl("PUT", "lcd/config/", gson.toJson(config), LcdConfigResponse.class).isOrientationForced();
+    }
+
+    public FreeboxSystemConfig getSystemConfig() throws FreeboxException {
+        return executeUrl("GET", "system/", null, SystemConfigResponse.class);
+    }
+
+    public boolean isInLanBridgeMode() throws FreeboxException {
+        return executeUrl("GET", "lan/config/", null, LanConfigResponse.class).isInBridgeMode();
+    }
+
+    public List<FreeboxLanHost> getLanHosts() throws FreeboxException {
+        List<FreeboxLanHost> hosts = new ArrayList<>();
+        List<FreeboxLanInterface> interfaces = executeUrl("GET", "lan/browser/interfaces/", null,
+                LanInterfacesResponse.class);
+        if (interfaces != null) {
+            for (FreeboxLanInterface lanInterface : interfaces) {
+                if (lanInterface.getHostCount() > 0) {
+                    List<FreeboxLanHost> lanHosts = getLanHostsFromInterface(lanInterface.getName());
+                    if (lanHosts != null) {
+                        hosts.addAll(lanHosts);
+                    }
+                }
+            }
+        }
+        return hosts;
+    }
+
+    private List<FreeboxLanHost> getLanHostsFromInterface(String lanInterface) throws FreeboxException {
+        return executeUrl("GET", "lan/browser/" + encodeUrl(lanInterface) + "/", null, LanHostsResponse.class);
+    }
+
+    public FreeboxPhoneStatus getPhoneStatus() throws FreeboxException {
+        return executeUrl("GET", "phone/?_dc=1415032391207", null, PhoneStatusResponse.class).get(0);
+    }
+
+    public List<FreeboxCallEntry> getCallEntries() throws FreeboxException {
+        return executeUrl("GET", "call/log/", null, CallEntryResponse.class);
+    }
+
+    public List<FreeboxAirMediaReceiver> getAirMediaReceivers() throws FreeboxException {
+        return executeUrl("GET", "airmedia/receivers/", null, AirMediaReceiversResponse.class, true, true, false);
```

Comment:
Was there a reason for not using the `org.eclipse.smarthome.io.net.http.HttpRequestBuilder` instead of `org.eclipse.smarthome.io.net.http.HttpUtil`? The builder would make code more readable when there are a lot of parameters.